### ISSUE TITLE
Unable to use with version of Ansible above 2.13

### ISSUE THREAD

**Issue by mfortin** [ansible-lockdown/AMAZON2-CIS#22]
**Describe the Issue**
Using ansible 2.16.3 there is an error while execuiting this role

command/shell - remove deprecated warn module param - [Ansible 2.14.0 Release Notes](https://github.com/ansible/ansible/blob/v2.14.0/changelogs/CHANGELOG-v2.14.rst)
Deprecation announcement - [Ansible 2.11 Release Notes](https://github.com/ansible/ansible/blob/v2.11.0/changelogs/CHANGELOG-v2.11.rst#L295)

**Expected Behavior**
No errors

**Actual Behavior**
```
TASK [AMAZON2-CIS : 1.2.2 | AUDIT | Ensure package manager repositories are configured | Get repo list] ***
fatal: [10.0.0.2]: FAILED! => changed=false
  msg: 'Unsupported parameters for (ansible.legacy.command) module: warn. Supported parameters include: _raw_params, _uses_shell, argv, chdir, creates, executable, expand_argument_vars, removes, stdin, stdin_add_newline, strip_empty_ends.'
```

**Control(s) Affected**

* 1.1.22
* 1.2.2
* 3.1.2
* 6.2.1
* handlers
* pre/post rememediation audit

**Environment (please complete the following information):**
 - Ansible Version: 2.16.3 
 - Host Python Version: Python 3.7.6
 - Ansible Server Python Version: Python 3.11.7
 - branch: devel
 - Additional Details: `warn: false` was deprecated

**Additional Notes**
Module FQCN are not used in the role

**Possible Solution**
Removing warn: false from commands/shell ansible tasks.

**Comment by danilo-lopes** [comment]
Is this project active ?

**Comment by uk-bolly** [comment]
hi @danilo-lopes 

Yes this is still active, i've just released v3.0.0 to the subscribers last week. Its not active in the community as much it seems.

Thanks

uk-bolly

**Comment by mfortin** [comment]
#24 addressed the compatibility issues with latest Ansible CLI
